plugins/woocommerce/composer.json

```diff
@@ -32,10 +32,12 @@
 		"automattic/jetpack-changelogger": "^3.3.0",
 		"bamarni/composer-bin-plugin": "^1.4",
 		"dms/phpunit-arraysubset-asserts": "^0.4.0",
+		"mockery/mockery": "1.6.6",
 		"phpunit/phpunit": "^9.0",
 		"sebastian/comparator": "^4.0",
 		"yoast/phpunit-polyfills": "^2.0",
-		"woocommerce/monorepo-plugin": "*"
+		"woocommerce/monorepo-plugin": "*",
+		"wp-hooks/generator": "0.9.0"
```

Comment:
Just wondering for sanity check, where/why these new composer deps turned up here?

Reply:
They came from `plugins/woocommerce-blocks/composer.json` since all of that PHP code is now included in Core. For instance, Mockery is needed for the PHPUnit tests to run.